<@dalias> basically linux/ptrace.h conflicts with sys/user.h on some archs
<@dalias> and linux/ptrace.h is utterly useless because sys/ptrace.h defines everything needed

diff --git a/ptrace.h b/ptrace.h
index d8af976..425b1e0 100644
--- a/ptrace.h
+++ b/ptrace.h
@@ -52,8 +52,6 @@ extern long ptrace(int, int, char *, long);
 # define ptrace_peeksiginfo_args XXX_ptrace_peeksiginfo_args
 #endif
 
-#include <linux/ptrace.h>
-
 #ifdef HAVE_STRUCT_IA64_FPREG
 # undef ia64_fpreg
 #endif

